autogen/safer-cluster/variables.tf.tmpl

```diff
@@ -26,12 +26,6 @@ variable "name" {
   description = "The name of the cluster"
 }
 
-variable "description" {
```

Comment:
nit: It would be good to keep this and plumb it through

Reply:
Done - That was probably less work than removing it in the first place.